Data Analysis

majdadel20 opened the pull request.

majdadel20 committed:
Data Analysis

oleksandr-maksymikhin commented:
Hello, Majd. Could you please check the CI Checks / ls-lint (pull_request) issue. It says that you are not allowed to use capital letters to name files. Regarding the second issue CI Checks / markdown-lint Yevgeniia knows better how to resolve it.

majdadel20 committed:
edit the error

majdadel20 commented:
> Hello, Majd. Could you please check the CI Checks / ls-lint (pull_request) issue. It says that you are not allowed to use capital letters to name files. Regarding the second issue CI Checks / markdown-lint Yevgeniia knows better how to resolve it.

I fixed the first issue and changed the file name. Thanks, olexander

Yevheniia-Rudenko committed:
Add markdownlint config to disable formatting checks
